src/core/net/netif.hpp

```diff
@@ -239,6 +239,30 @@ class Netif : public InstanceLocator
      */
     otError RemoveUnicastAddress(const NetifUnicastAddress &aAddress);
 
+    /**
+     * This method adds an internal (to OpenThread) unicast address to the network interface.
+     *
+     * @param[in]  aAddress  A reference to the unicast address.
+     *
+     * @retval OT_ERROR_NONE          Successfully added (or updated) the unicast address.
+     * @retval OT_ERROR_INVALID_ARGS  The address indicated by @p aAddress is an internal address.
```

Comment:
From the implementation, this should be "... is link-local address or an external address."